velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -319,4 +319,20 @@ struct DayOfWeekFunction : public InitSessionTimezone<T>,
   }
 };
 
+template <typename T>
+struct DateDiffFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
```

Comment:
Would you check https://github.com/facebookincubator/velox/pull/6476 and https://github.com/facebookincubator/velox/issues/6428 to see if the issue there affects this function as well. CC: @kgpai

Reply:
Thanks, I think this PR also has this issue. we can wait @gggrace14  to fix it. https://github.com/facebookincubator/velox/issues/6428#issuecomment-1714511888

Reply:
Hi, @mbasmanova , sorry for bother you. Since https://github.com/facebookincubator/velox/pull/6550 was merged. Can you review this PR again?